### ISSUE TITLE
custom_link doesn't respect new_window value

### ISSUE THREAD

**Issue by fellipeh** [farridav/django-jazzmin#421]
Hi,

I have this code:
```    "custom_links": {
        "relatorio": [
            {
                "name": "Presença Quantitativo",
                "url": "presencao_quantitativo",
                "new_window": True,
                "icon": "fas fa-calendar",
                "permissions": ["presencao.quantitativo"],
            }
        ]
    },
```

It's supposed to open that link into a new window, but it doesn't. Am I doing something wrong?

**Comment by farridav** [comment]
Nope, it seems there is a bug, we are not adding the `target="_blank"` for links in the side menu, see https://github.com/farridav/django-jazzmin/blob/master/jazzmin/templates/admin/base.html#L223

We need to add a `{% if model.new_window %}target="_blank"{% endif %}` into the `<a>` tag ... 

If you want to make a PR for this, ill gladly review it, if not, ill add it to my list of things to do (I'm a little stretched at the moment)
